recsys/recsys_tutorial.py

```diff
@@ -172,17 +172,17 @@ def polarity_negative(x):
 L_train = applier.apply(df_train)
 label_model = LabelModel(cardinality=2, verbose=True)
 label_model.fit(L_train, n_epochs=5000, seed=123, log_freq=20, lr=0.01)
-Y_train_preds = label_model.predict(L_train)
+Y_preds_train = label_model.predict(L_train)
```

Comment:
I've used `preds_train` in the slicing tutorial, because `Y` seems redundant — does that make sense here?